### ISSUE TITLE
Signup/Login with email

### ISSUE THREAD

**Issue by jiakeliu** [gunnertech/thumbwar-ios-client#100]
@CodySwannGT does our backend support sign-up through email?

**Comment by CodySwannGT** [comment]
Nope. Facebook connect was decided as the only way to sign up ala Tinder.

We originally supported it in the first app we built, but the group decided to move to Facebook connect only, so it was removed for the second app.

**Comment by CodySwannGT** [comment]
Just curious - Why would you want to allow signup via email? 99.999999% of people have Facebook

**Comment by jiakeliu** [comment]
I can understand Tinder's design decision for using Facebook Login exclusively, because a dating network requires authenticity. I'm not sure if Thumbwar requires the same level of authenticity. Since this is outside the scope of our MVP, I'm moving it to Future Consideration and wait for user feedback from the MVP release.

**Comment by CodySwannGT** [comment]
So basically, you're in favor of letting someone create a bunch of fake
accounts ala twitter?

I see that as the only thing you GAIN from allowing people to register with
an email address in addition to Facebook.

On Sun, May 10, 2015 at 9:05 PM, Jiake Liu <notifications@github.com> wrote:

> I can understand Tinder's design decision for using Facebook Login
> exclusively, because a dating network requires authenticity. I'm not sure
> if Thumbwar requires the same level of authenticity. Since this is outside
> the scope of our MVP, I'm moving it to Future Consideration and wait for
> user feedback from the MVP release.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gunnertech/thumbwar-ios-client/issues/100#issuecomment-100726762>
> .
>



-- 
 Cody Swann - CEO, Gunner Technology
T: 213.915.4083 | cody@gunnertech.com | www.gunnertech.com
<http://www.gunnertech.com/>Skype: cody.swann

*CONFIDENTIALITY NOTICE: This email message, together with any documents,
files and/or email messages attached to it, is intended for the sole use of
the individual or entity to whom it is addressed, and may contain
information that is legally privileged, confidential and restricted from
disclosure.  If you are not the intended recipient, or responsible for
delivery to that person, you are hereby notified that any dissemination or
copying of this communication is STRICTLY PROHIBITED.  In such case please
notify the sender by reply email and delete this message without reading,
printing or saving.*

**Comment by jiakeliu** [comment]
Definitely not in favor of bunch of fake accounts, and I realize that's the pitfall for email registration, although there are solutions to mitigate this. But more importantly there are some positives we also need to consider, such as people who are turned off by having to disclose their personal identity (ala sign-in via Facebook) just to see the app content. I've asked a few friends who are in our target demographic, and they all don't feel comfortable about the Facebook login wall.

Maybe a happy medium is to allow guests to view content, but they have to login via Facebook to participate. Something to think about.

**Comment by CodySwannGT** [comment]
Yeah. Everything you're mentioning is stuff we discussed at length.

I'm not here to tell you you're wrong or anything like that. Just catching
you up on many discussions we had.

The concession was:

1) Facebook Connect allows you to require different levels of info. We
would go for the least necessary
2) We wanted people who WERE comfortable being public.

Something we didn't consider was the read-only, non logged in user. I
personally like that idea, so people can see what it is before they sign up

On Sun, May 10, 2015 at 9:17 PM, Jiake Liu <notifications@github.com> wrote:

> Definitely not in favor of bunch of fake accounts, and I realize that's
> the pitfall for email registration, although there are solutions to
> mitigate this. But more importantly there are some positives we also need
> to consider, such as people who are turned off by having to disclose their
> personal identity (ala sign-in via Facebook) just to see the app content.
> I've asked a few friends who are in our target demographic, and they all
> don't feel comfortable about the Facebook login wall.
>
> Maybe a happy medium is to allow guests to view content, but they have to
> login via Facebook to participate. Something to think about.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gunnertech/thumbwar-ios-client/issues/100#issuecomment-100728373>
> .
>



-- 
 Cody Swann - CEO, Gunner Technology
T: 213.915.4083 | cody@gunnertech.com | www.gunnertech.com
<http://www.gunnertech.com/>Skype: cody.swann

*CONFIDENTIALITY NOTICE: This email message, together with any documents,
files and/or email messages attached to it, is intended for the sole use of
the individual or entity to whom it is addressed, and may contain
information that is legally privileged, confidential and restricted from
disclosure.  If you are not the intended recipient, or responsible for
delivery to that person, you are hereby notified that any dissemination or
copying of this communication is STRICTLY PROHIBITED.  In such case please
notify the sender by reply email and delete this message without reading,
printing or saving.*
